### ISSUE TITLE
Warnings even when using the 'Recommended querier timeout'

### ISSUE THREAD

**Issue by LouisAtGH** [troglobit/pimd#237]
I noticed the following warnings in the system log:
Dec 18 13:33:19 pfSense pimd[24]: Recommended querier timeout = Robustness x query-interval + response-time / 2 = 3 x 12 + 10 / 2 = 41

Which is in two regards strange:
- the default used to be 42 and
- given this warning I changed it to 41

And the warning is still there and repeating with some frequency

After noticing this I checked the actual settings with pimctl show status, which confirmed that the query interval is the advised 41

[2.7.0-DEVELOPMENT][root@pfSense.lan]/root: pimctl show status
PIM Daemon Status
Elected BSR
    Address          : 0.0.0.0
    Expiry Time      : 0h0m40s
    Priority         : N/A
    Hash Mask Length : 30
Candidate BSR
    State            : Disabled
    Address          : 0.0.0.0
    Priority         : N/A
Candidate RP
    State            : Enabled
    Address          : 192.168.14.1
    Priority         : 0
    Holdtime         : 150 sec
Join/Prune Interval  : 60 sec
Hello Interval       : 30 sec
Hello Holdtime       : 105 sec
IGMP query interval  : 12 sec
IGMP querier timeout : 41 sec
SPT Threshold        : Enabled
SPT Mode             : packets
SPT Packets          : 0
SPT Interval         : 100 sec

**Comment by troglobit** [comment]
Yeah this should be changed to follow the RFC values for query and timeout (180 and 300 iirc).

Thank you for the reminder 👍
